waggle-dance-core/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
there should not be any log4j config file here please remove, this is created by the packaging module: https://github.com/ExpediaGroup/waggle-dance/blob/main/waggle-dance/src/main/resources/log4j2.xml